youtube_dl/extractor/youtube.py

```diff
@@ -388,6 +388,7 @@ class YoutubeIE(YoutubeBaseInfoExtractor):
                             (?:(?:www|no)\.)?invidiou\.sh/|
                             (?:(?:www|fi|de)\.)?invidious\.snopyta\.org/|
                             (?:www\.)?invidious\.kabi\.tk/|
+                            (?:www\.)?invidiou(?:s)?\.site/|
```

Comment:
1. Superfluous second group.
2. invidiou.site does not work.

Reply:
it appears that invidiou.site is down now.